packages/block/src/header.ts

```diff
@@ -56,6 +56,7 @@ export class BlockHeader {
   public readonly withdrawalsRoot?: Uint8Array
   public readonly dataGasUsed?: bigint
   public readonly excessDataGas?: bigint
+  public readonly beaconRoot?: Uint8Array
```

Comment:
```suggestion
  public readonly parentBeaconBlockRoot?: Uint8Array
```
to be one to one with eip and specs

Reply:
Hmm weird how I did not pick up this change to the EIP, thanks :smile: